src/apps/explorer/components/TransactionsTableWidget/index.tsx

```diff
@@ -60,6 +83,18 @@ export const TransactionsTableWidget: React.FC<Props> = ({ txHash }) => {
     return (): void => clearTimeout(timer)
   })
 
+  const onChangeTab = useCallback((tabId: number) => {
+    const newTabViewName = Object.keys(TAB_VIEW_ID).find((key) => TAB_VIEW_ID[key as TabViews] === tabId)
```

Comment:
I feel like the TAB_VIEW_ID is not really needed.

What about you instead create the enum without string keys.

See https://stackoverflow.com/a/55578603/1272513 for reference of what I mean

Reply:
Here if I didn't get it 🤔, 
if I create the *enum* without string like:

```typescript
enum TabView {
  GRAPH,
  ORDERS,
}
```

then I would need a way to get the `tabNumberId` from the **string Name** (`tab` [URL param](https://github.com/cowprotocol/explorer/blob/41-anchor-url/src/apps/explorer/components/TransactionsTableWidget/index.tsx#L42)) and `tabViewName` from the **identifier number** ([onChangeTab callback](https://github.com/cowprotocol/explorer/blob/41-anchor-url/src/apps/explorer/components/TransactionsTableWidget/index.tsx#L84)) to update the URL params.

Reply:
I've created this [ts playground](https://www.typescriptlang.org/play?#code/FAehAIEkDNwTwPYFdwFskGcAu5sEMAnHLACwFNwBLAOwBMyAPcaAhVcARiuuzL1vAJYABgA04AFaYcqPAGsKpCowAOAG0oBjLJXB4MGSgHNqqMtSzBzSdgBU8AIwBqlMgHdwAb2Dhf4AOIASgCCAAoAEuAAvJyiPn4A8oEAIgCigQDK0eAATMAAvsCgEPoYNjRG4KSUGFS1biR4OIgoRmQ4LGxV5OBIBGrAmgg8xI4Acnhm2QDkRgR4KiTTRWDgALJ4cA6K5ARk07V44G6bVQjgtOekTceUpMjEPZr6OtRG4g5IOHjUcCdw4mqtUuZHqjRwbgoJwsg2GGAQajIADo1AgjAAKezOVxuADaWHGk2RWAQAFUVCoyAQAML6MjogCUelqSGocmoCDc1GZ4CxLncAF0GSsIIisAcqgQ4HpwM8MBQOR4GlSdhQFNKQRhqNMcIwajgaN1lNQbLCRsdWG8sRMpjFphyCQSHMshjwEcjURi+TjcW5LUZrUSeaz2ZzufpeY5+W4hSLmWVUBVujU6sdwamhom3sxWOwlFVHFQBM81GoHHhNHIzdgCw5IAIYhwiq74YiUWjMVGfU767HisnaimGjc3Mg1AJtrgSXsBJRYJgk-nrOxaJQ9to1HBq6M67QAGK51Im9gxb3uJFBMLhZtw93tr1d9z4xz1g9sI82PtAA) explaining my thinking here

You can run to see the results and play around with it

One thing to keep in mind is that TS enums are mapped to 2 way data bindings on JS code.
So you can access the values both by key and value to get the opposite:

```ts
enum E {
  value = 0
}

E['value'] === 0 // === E.value
E[0] === 'value'
```

Reply:
Got it.
Thanks for the details!